tests/azure/vm_backup_test.go

```diff
@@ -0,0 +1,73 @@
+// +build slow
+
+package azure_test
+
+import (
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/NeowayLabs/klb/tests/lib/azure/fixture"
+)
+
+func backupVM(t *testing.T, f fixture.F, vmname string, prefix string) string {
+
+	res := execWithIPC(t, f, func(output string) {
+		f.Shell.Run(
+			"./testdata/backup_vm.sh",
+			vmname,
+			f.ResGroupName,
+			prefix,
+			f.Location,
+			output,
+		)
+	})
+	res = strings.TrimSpace(res)
+	return strings.Trim(res, "\n")
```

Comment:
Really? Did not know that \n is considered as a space on that case, thanks :-). I read on gopl that it would handle proper unicoding issues, that is why I used here.